.github/workflows/dists.yml

```diff
@@ -41,18 +41,22 @@ jobs:
         # pin to the latest supported python version
         python-version: "3.13"
         # x86 python needs to be available for the win32 wheel
-        architecture: ${{ ( matrix.os == 'windows-latest' && matrix.target == 'i686' ) && 'x86' || null }}
+        architecture: ${{ ( matrix.os == 'windows-2025' && matrix.target == 'i686' ) && 'x86' || null }}
+
+    - name: Add python lib to PATH
+      if: matrix.os == 'windows-2025'
+      run: Add-Content $env:GITHUB_PATH "${{ env.pythonLocation }}\lib"
```

Comment:
something broke upstream in the past two weeks. this is now necessary to get python313.lib discovered by maturin

Reply:
alright, we're green with 3.13t 🎉

turns out we don't need this `Add python lib to PATH` step. we need an explicit 3.13t distribution installed instead.